handbook/marketing/new-workshop.sh

```
@@ -0,0 +1,261 @@
+      --label "$CHILD_LABELS,$NEW_LABEL" \
+      "${OPTIONAL_ARGS[@]}")
+
+    if [ -n "$CHILD_URL" ]; then
+        CHILD_NUM=$(echo "$CHILD_URL" | awk -F/ '{print $NF}')
+
+        CHILD_NODE_ID=$(gh api graphql -f query='
+          query($owner:String!, $repo:String!, $number:Int!) {
+            repository(owner:$owner, name:$repo) {
+              issue(number:$number) { id }
+            }
+          }' -f owner="$ORG" -f repo="$REPO" -F number="$CHILD_NUM" --jq '.data.repository.issue.id')
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)